acct.bionimbus.org/manifest.json

```diff
@@ -36,9 +35,6 @@
   "peregrine": {
     "sidecar": "True"
   },
-  "jupyterhub": {
```

Comment:
this needs to stay

Reply:
Jupyterhub is an ancient component that is no longer being used.
This should be removed from all manifests.
cc'ing @atharvar28 to confirm (Hey Atharva, it is safe to remove that, right?)